### PR TITLE
GraphQLCall, Get Full and Final Predictions Examples

### PR DESCRIPTION
**Pull request by nateshim-indico**
If there are other examples that come to mind let me know. For now, I am planning on doing DocumentExtraction and UserMetrics/Reports in a separate pr

### CODE REVIEW

File: Examples/GetFinalPredictions/Program.cs [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using IndicoV2;
+using Newtonsoft.Json.Linq;
+
+namespace Examples
+{
+    /// <summary>
+    /// Returns final output. Does not include pre_review and post_review predictions.
```
Comment (Scott771) [inline Examples/GetFinalPredictions/Program.cs:11]: This definitely returns predictions "post_review" and nothing else? vs. it returning only the raw results?

File: Examples/GetFullPredictions/Program.cs [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+            File.ReadAllText(Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.UserProfile),
+                "indico_api_token.txt"));
+
+        public static async Task Main()
+        {
+            var client = new IndicoClient(GetToken(), new Uri("https://app.indico.io"));
+
+            var dataSets = await client.DataSets().ListAsync();
+
+            var workflows = await client.Workflows().ListAsync(dataSets.First().Id);
```
Comment (Scott771) [inline Examples/GetFullPredictions/Program.cs:25]: I'm personally of the opinion that you write a workflowID into the example vs. include some methods to get a workflow ID that no one would ever actually use-- but curious what Meg thinks (obviously makes testing them impossible)

Comment (goatrocks) [reply Examples/GetFullPredictions/Program.cs:25]: I believe we typically either show how to get a workflow id from a dataset id or just hardcode it in the python examples -- so following that here would be good. People can hand change the workflow id or dataset id if they need to. 

adding comments explaining what is happening or pointing to where to find workflow id can help, too.

File: Examples/GetFinalPredictions/Program.cs [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    /// </summary>
+    internal class GetFinalPredictions
```
Comment (goatrocks) [inline Examples/GetFinalPredictions/Program.cs:13]: these can be public instead of internal

File: Examples/GraphqlCall/Program.cs [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    /// </summary>
+    internal class GraphQLCall
+    {
+        private static string GetTokenPath() =>
+            Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.UserProfile),
+                "indico_api_token.txt");
+
+        public static async Task Main()
+        {
+            var config = new IndicoConfig(GetTokenPath(), "app.indico.io");
```
Comment (goatrocks) [inline Examples/GraphqlCall/Program.cs:21]: Ideally we could wrap these in a V2 call/interface like the other straggler V1 calls, so we can rewrite it in the future without also changing the interface.

**Review by goatrocks — CHANGES_REQUESTED** [review]
If it's possible to wrap the existing graphql call in a new interface in V2, that would be awesome.

File: Examples/GetFinalPredictions/Program.cs (hunk 1 above)
Comment (nateshim-indico) [reply Examples/GetFinalPredictions/Program.cs:11]: it returns raw results (pre_review)

**Review by goatrocks — APPROVED** [review]

**Pull request merged**